### PR TITLE
Merge upstream changes

### PR DESCRIPTION
**Pull request by mdholloway**
Needed for upgrading deps to mapnik 3.7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: node_js
 
 node_js:
-- '0.10'
 - '4'
```
Comment (nyurik) [inline .travis.yml]: might as well kill `4`?

**Comment by thesocialdev** [comment]
LGTM. I will wait for someone else's inputs, but I am inclined to merge.

**Review by nyurik — APPROVED** [review]
other than perhaps dropping node 4 support in Travis, and adding v8, seems to be good

**Review by thesocialdev — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (mdholloway) [reply .travis.yml]: Done!

**Pull request merged**